### PR TITLE
Add Claire Zuliani profile

### PR DESCRIPTION
**Pull request by clairezed**
No picture of Epinal yet in the main repo :) I'll be glad to provide you with one if you want !

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -14081,6 +14082,55 @@ var data = {
...
+        'https://d33wubrfki0l68.cloudfront.net/dddd2605dce105601dcba9587f14633175792c58/82039/assets/images/avatar.png',
+      location: 'Epinal',
+      cities: ['Epinal', 'Nancy', 'Metz'],
+      contacts: {
+        twitter: 'ZooletteDesBois',
+        mail: 'clairezuliani@gmail.com',
+      },
+      websites: [
+        {
+          name: 'Portfolio',
```
Comment (dadoonet) [inline baggers.js]: Not supported name. See the list in the README.

File: baggers.js [hunk 2]
```diff
@@ -14081,6 +14082,55 @@ var data = {
...
+          url: 'https://www.clairezuliani.com',
+        },
+        {
+          name: 'Carnet',
```
Comment (dadoonet) [inline baggers.js]: Not supported name. See the list in the README.

**Review by dadoonet — CHANGES_REQUESTED** [review]
Left some comments. Would be great if you have an image of Epinal ;)

Send your PR there https://github.com/brownbaglunch/BrownBagLunch/tree/gh-pages/img/villes

File: baggers.js (hunk 2 above)
Comment (clairezed) [reply baggers.js]: oh, sorry ! I hadn't noticed the restricted list !

**Review by clairezed — COMMENTED** [review]
Update websites to cop with authorized websites list

**Review by dadoonet — CHANGES_REQUESTED** [review]
Note that you can have more than one Web link if you wish so if you want to add the second one, no worries.

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Bienvenue !

**Pull request merged**